code/game/mecha/mecha.dm

```diff
@@ -1320,7 +1320,7 @@
 /obj/mecha/proc/operation_allowed(mob/living/carbon/human/H)
 	if(!ishuman(H))
 		return 0
-	for(var/ID in list(H.get_active_hand(), H.wear_id, H.belt))
+	for(var/ID in list(H.get_active_hand(), H.wear_id, H.belt, H.wear_pda))
```

Comment:
This is technically fine because otherwise it'd cause a much larger refactor of how mech code access works. However, can you change the line on 1330 to match the changes in this list?